drivers/pwm/pwm_ambiq_ctimer.c

```diff
@@ -127,8 +127,8 @@ static void start_clock(uint32_t clock_sel)
 }
 
 static int ambiq_ctimer_pwm_set_cycles(const struct device *dev, uint32_t channel,
-				      uint32_t period_cycles, uint32_t pulse_cycles,
-				      pwm_flags_t flags)
+				       uint32_t period_cycles, uint32_t pulse_cycles,
+				       pwm_flags_t flags)
```

Comment:
This should be corrected in the commit that changes `timer` to `ctimer` in function names, not in the one that introduces assertions.
There are much more such misplaced corrections in this PR.